timestamp-api/build.gradle

```diff
@@ -2,6 +2,8 @@ apply from: "../gradle/publish-jars.gradle"
 apply from: "../gradle/shared.gradle"
 
 dependencies {
-  compile 'javax.ws.rs:javax.ws.rs-api:2.0.1'
-  compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations'
+    compile 'javax.ws.rs:javax.ws.rs-api:2.0.1'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations'
+
+    testCompile group: 'org.assertj', name: 'assertj-core'
```

Comment:
Yep. The dependency lock only seems to apply to main (compile/runtime) configurations, not test ones.